Dockerfile

```diff
@@ -1,15 +1,11 @@
 # syntax=docker/dockerfile:1.4
-FROM node:14.21.1-alpine3.15 AS base
+FROM node:18.15.0-alpine3.17 AS base
+ENV HUSKY 0
```

Comment:
Whats the usage of these

Reply:
Husky is a pre-commit and setup hooks module. We're trying to disable it with this environment variable.